perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/ServicesManagerBlImpl.java

```diff
@@ -271,6 +274,52 @@ public void deleteService(PerunSession sess, Service service) throws RelationExi
 		getPerunBl().getAuditer().log(sess, new ServiceDeleted(service));
 	}
 
+	/*
+	 * Tables with reference to service:
+	 *   - service_required_attrs
+	 *   - service_denials
+	 *   - resource_services
+	 *   - facility_service_destinations
+	 *   - service_service_packages
+	 *   - tasks
+	 *   - authz
+	 */
+	@Override
+	public void deleteService(PerunSession sess, Service service, boolean forceFlag) throws RelationExistsException, ServiceAlreadyRemovedException {
```

Comment:
You can rewrite the original method deleteService() and use the flag there. It will prevent duplicates of code.

Reply:
Done.